src/kdc/main.c

```diff
@@ -955,6 +955,21 @@ int main(int argc, char **argv)
 
     initialize_kdc5_error_table();
 
+    retval = krb5_get_profile(kcontext, &profile);
```

Comment:
Same way as max_dgram_reply_size is read; there is no categorical difference between the two parameters.  (Functionally, it makes no difference.)